package.json

```diff
@@ -21,7 +22,10 @@
     "lint-fix:css": "stylelint \"**/**.{scss,vue}\" --fix",
     "lint-fix:md": "markdownlint \"**/***.md\" --ignore-path .gitignore --ignore LICENSE.md --fix",
     "lint": "npm-run-all lint:js lint:css lint:md",
-    "lint:fix": "npm-run-all lint-fix:*",
+		"lint:fix": "npm-run-all lint-fix:*",
+		"release:major": "npx release major",
+		"release:minor": "npx release minor",
+		"release:patch": "npx release patch",
```

Comment:
I don't see `release` as a dependency. Is it included by some other means? Also if it is installed as a dev dependency, I don't think we'd need to use `npx`.